### ISSUE TITLE
mobilenet structure omitts last pooling layer

### ISSUE THREAD

**Issue by zhenglaizhang** [deepinsight/insightface#40]
Have a check at the mobilenet structure, and most of the weights are in the final FC layer:
```
pre_fc1_weight parameter size=25690112, shape=(512L, 50176L)
```
and looking at the code, the last pooling layer is omitted (![mobilenet](https://raw.githubusercontent.com/joshua19881228/my_blogs/master/Computer_Vision/Reading_Note/figures/Reading_Note_20170719_MobileNet_1.png)).
https://github.com/deepinsight/insightface/blob/f1cd542e0f0413ed0e06723e5c49593e3e468825/src/symbols/fmobilenet.py#L75-L80

Is this by design? Ignoring the last pooling layer leads to much larger model size -:(

**Comment by nttstar** [comment]
please refer to network structure section in our paper.

**Comment by zhenglaizhang** [comment]
@nttstar thanks for the info, I will read carefully the paper.

**Comment by zhenglaizhang** [comment]
@nttstar Hi, I tried the mobilenetv1 with `D` as version output, and started to train 12 hours ago, with 128 batch size, but found the test accuracy is still around 0.5.

Have you guys tried to train with such settings? maybe I need to tune other hyperparameters...

**Comment by nttstar** [comment]
@zhenglaizhang We all use 'E' in our recent experiments.

**Comment by mlinxiang** [comment]
@zhenglaizhang i also has this question, and in the code last conv directly connet with mx.sym.FullyConnected, is this func will do flatten inner?
